sessions/automated-testing.qmd

```diff
@@ -12,7 +12,13 @@
 
 **Session objectives**:
 
--   Deploy a GitHub Action that runs a continuous integration workflow
+The overall objective for this session is to:
+-   Explain what *automated testing* means when developing packages, 
+    and deploy simple GitHub Actions to automate tests
```

Comment:
"deploy" is a fairly "jargon" term and doesn't precisely say what they will do. Something like "create and upload".

Reply:
@lwjohnst86 Hm, I'm not sure that "upload" covers what I want to say. How about:
```suggestion
    and create simple GitHub Actions that will automatically run tests when new code is pushed to GitHub
```

Reply:
Fantastisk! 🎉